### ISSUE TITLE
1.7.0.0 error report on startup

### ISSUE THREAD

**Issue by brennansingular** [SingularSound/openbbm#130]
@GoranRista  reported: 
Every time the user opens the BB Manager it throws errors for songs there were edited previous day with AP.
"Processing errors: [song name] The song was recovered..."
The error log is:
"Please make sure every song part has a Main Drum Loop."
Every song part has main loops, so this error should not be happening.

![BB Error](https://user-images.githubusercontent.com/62663388/94489209-0aaf5000-01b2-11eb-973c-c0f8d857aa1a.jpg)

This was on my computer after start up:
![Screenshot (40)](https://user-images.githubusercontent.com/62663388/94489202-084cf600-01b2-11eb-9eb4-a12f533584a3.png)

**Comment by brennansingular** [comment]
@JulissaDantes I don't believe the same behavior is present in 1.8.0.0, did you apply a fix for this?

**Comment by JulissaDantes** [comment]
@brennansingular fixes were made on the warning workflow, but I didnt fix this bug because I never experienced it, if the latest version is not presenting the warning maybe one of the changes might have fix it.  I will be checking this out

**Comment by JulissaDantes** [comment]
@brennansingular can you upload the file here? I tried exporting songs and re importing them but could not see this happening

**Comment by brennansingular** [comment]
[New_Song_julissa.zip](https://github.com/SingularSound/openbbm/files/5337227/New_Song_julissa.zip)
Here is the song that came up in the error
@JulissaDantes

**Comment by JulissaDantes** [comment]
![image](https://user-images.githubusercontent.com/8561085/95920169-e9875b80-0d7c-11eb-8b3e-5c41da6f0059.png)
This is the song I imported, but it does not look like the songs from your screenshots, still not getting the error, Im I supposed to do something before closing the program to open it again?

**Comment by brennansingular** [comment]
I hadn't done anything differently, will have to keep an eye out for this, possibly it is fixed.

**Comment by JulissaDantes** [comment]
New user input: This seems to be happening with whatever song is being edited last before closing the project.
